src/backend/distributed/utils/metadata_cache.c

```diff
@@ -2221,6 +2253,20 @@ GetLocalGroupId(void)
 
 
 /*
+ * RegisterLocalGroupIdCacheCallbacks registers the callbacks required for the
+ * worker node cache.  It's separate from InitializeWorkerNodeCache so the
+ * callback can be registered early, before the tables exist.
```

Comment:
Comment is copied from `RegisterWorkerNodeCacheCallbacks` and needs to be adapted.